src/stylesheets/core/mixins/_at-media.scss

```diff
@@ -3,15 +3,16 @@
   $orig-bp: $bp;
   $quoted-bp: quote($bp);
   $our-breakpoints: map-deep-get($system-properties, breakpoints, standard);
-  @if map-has-key($our-breakpoints, $quoted-bp){
+  @if map-has-key($our-breakpoints, $quoted-bp) {
     @if $theme-respect-user-font-size {
       $bp: rem-to-user-em(map-get($our-breakpoints, $quoted-bp));
-    } @else {
+    }
+    @else {
```

Comment:
@uswds/core Since we disable this for `@else`, do we want to put these back to how they were?

Reply:
or should we leave them for consistency, looking through some instances here, I see we wrote them both ways.

Reply:
Maybe I should work on making them consistent in a subsequent PR

Reply:
Doesn't matter to me, I think we have a lot more cases of having the `else`s on another line, for what it's worth

Reply:
And call this work done

Reply:
That works for me, approved!